rdflib/tools/chunk_serializer.py

```diff
@@ -0,0 +1,136 @@
+"""
+This file provides a single function `serialize_in_chunks()` which can serialize a
+Graph into a number of NT files with a maximum number of triples or maximum file size.
+
+There is an option to preserve any prefixes declared for the original graph in the first
+file, which will be a Turtle file.
+"""
+
+import os
+from pathlib import Path
+
+try:
+    from rdflib import Graph, Literal
+except Exception:
+    import sys
+
+    sys.path.insert(0, str(Path(__file__).parent.parent.parent.absolute()))
+    from rdflib import Graph, Literal
+
+__all__ = ["serialize_in_chunks"]
+
+
+def serialize_in_chunks(
+    g: Graph,
+    max_triples: int = 10000,
+    max_file_size_kb: int = None,
+    file_name_stem: str = "chunk",
+    output_dir: Path = Path(__file__).parent,
+    first_file_contains_prefixes: bool = False,
```

Comment:
I think it may be best to just have this:

```suggestion
    write_prefixes: bool = False,
```

And then use `_prefixes.ttl` instead for the file name.